packages/preview-middleware-client/src/adp/controllers/AddFragment.controller.ts

```diff
@@ -216,8 +225,9 @@ export default class AddFragment extends BaseDialog {
 
             this.model.setProperty('/fragmentList', fragments);
         } catch (e) {
-            MessageToast.show(e.message);
-            throw new Error(e.message);
+            const error = getError(e);
```

Comment:
why don't you use `handleErrror` here

Reply:
Fixed in [39505b0](https://github.com/SAP/open-ux-tools/pull/1918/commits/39505b01b7d9cb4ccdc43b1ab21387e06780dc7a)